Source/ThirdParty/zip.js

```diff
@@ -779,6 +779,13 @@ define(['../Core/buildModuleUrl'], function(buildModuleUrl) {
 		};
 	}
 
+	function getWorkerScriptsPath(zip) {
```

Comment:
it would be a lower-impact change to define `workerScriptsPath` as a ES5 getter on `obj.zip`, and would still let you delay the URL-building.